src/main/resources/db/migration/general/V4_4__offender_case_note_user_id.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE offender_case_note
+    ADD author_user_id VARCHAR(36) NULL;
+
+UPDATE offender_case_note
+SET author_user_id = author_username;
```

Comment:
note setting the `user_id` to be `username` for existing rows

Reply:
👍